source/Window.cpp

```diff
@@ -41,14 +40,20 @@ void Window::setConfig(radix::Config &config){
 }
 
 void Window::initGl() {
+  // Parse OpenGL Version
   gl::OpenGL::initialize();
-  const int glver = gl::OpenGL::version(), glmaj = glver / 10, glmin = glver % 10;
-  const std::string versionString = std::to_string(glmaj) + '.' + std::to_string(glmin);
+  const int glversion = gl::OpenGL::version();
+  const int glmajor = glversion / 10;
+  const int glminor = glversion % 10;
+  // Convert OpenGL Version to string
+  char versionString[8];
```

Comment:
@wow2006 This is the same problem of comments describing the what and not the why. You can explain this code a lot better if you move the code into a method getOpenGlVersionString() for example. It is a good piece of code to put in a black box since it is unlikely to change but if it has to change the small resulting method will be easy to understand.